pytest_fixtures/component/satellite_auth.py

```diff
@@ -293,7 +308,19 @@ def enroll_idm_and_configure_external_auth(session_target_sat):
             raise CLIReturnCodeError(result.status, result.stderr, 'Failed to enable ipa client')
 
 
-@pytest.fixture(scope='session')
+@pytest.mark.external_auth
+@pytest.fixture(scope='module')
+def module_enroll_idm_and_configure_external_auth(module_target_sat):
+    enroll_idm_and_configure_external_auth(module_target_sat)
+
+
+@pytest.mark.external_auth
+@pytest.fixture
+def func_enroll_idm_and_configure_external_auth(target_sat):
```

Comment:
we specify fixture scope in name only if its module scoped, so I feel we should keep this consistent
```suggestion
def enroll_idm_and_configure_external_auth(target_sat):
```